backend/habragasolina/src/main/java/bo/edu/usfa/gasolina/habragasolina/Service/UserService.java

```diff
@@ -0,0 +1,76 @@
+package bo.edu.usfa.gasolina.habragasolina.Service;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Service;
+import org.springframework.web.server.ResponseStatusException;
+
+import bo.edu.usfa.gasolina.habragasolina.Entities.User;
+import bo.edu.usfa.gasolina.habragasolina.Repository.GasStationRepository;
+import bo.edu.usfa.gasolina.habragasolina.Repository.UserRepository;
+
+@Service
+public class UserService {
+    
+    private final UserRepository userRepository;
+    
+    private final GasStationRepository gasStationRepository;
+
+    public UserService(UserRepository userRepository, GasStationRepository gasStationRepository) {
+        this.userRepository = userRepository;
+        this.gasStationRepository = gasStationRepository;
+    }
+    
+    public User saveUser(User user) {
+
+        //validations
+        if(user.getPassword().length() <= 6){
+            throw new ResponseStatusException(HttpStatus.NOT_FOUND, "Password must be longer or equal than 6 characters");
```

Comment:
Why a Not Found exception? We can fix the ticket documentation to return a 400 - Bad Request